File: 1_Multiples_of_3_and_5.py
# Created December, 2017

a, b, c, s = 3, 5, 15, 0
while a < 1000:
    s += a
    a += 3
while b < 1000:
    s += b
    b += 5
while c < 1000:
    s -= c
    c += 15
print(s)

File: 1_Multiples_of_3_and_5.c
// Created December, 2017

#include <stdio.h>

int main() {
    int a = 3, b = 5, c = 15, sum = 0;
    while (a < 1000) {
        sum += a;
        a += 3;
    }
    while (b < 1000) {
        sum += b;
        b += 5;
    }
    while (c < 1000) {
        sum -= c;
        c += 15;
    }
    printf("%d\n", sum);
}
